R/render.r

```diff
@@ -162,6 +170,21 @@ check_open_graph <- function(og) {
   og[intersect(supported_fields, names(og))]
 }
 
+get_bs_version <- function(pkg = ".") {
+  pkg <- as_pkgdown(pkg)
+
+  template <- pkg$meta[["template"]]
+
+  if (!is.null(template$bootstrap)) {
+    if (!template$bootstrap %in% c(3, 4)) {
+      stop("The Bootstrap version `bootstrap` has to be 3 or 4.", call. = FALSE)
+    }
```

Comment:
I'd make this an else statement so the overall structure is like:

```R
if (has_parameter) {
  if (parameter_ok) {

  } else {
   abort(....)
 } else {
  3
}
```

Then it's more clear that the else clause if is very simple, so I think it's clearer to flip the order so that the simplest case becomes an early exit:

```R
if (is.null(template$bootstrap)) {
  return(3)
}
```